resources/crm-local-develop-environment/cluster/redis.sh

```
@@ -0,0 +1,17 @@
+PASSWORD=$(expr $1)
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)